PIL/Image.py

```diff
@@ -599,7 +635,7 @@ def _dump(self, file=None, format=None, **options):
             self.save(file, format, **options)
         return file
 
-    def __eq__(self, other):
+    def __eq__(self, other):  # type: ignore # type: (Image) -> bool
         return (self.__class__.__name__ == other.__class__.__name__ and
```

Comment:
The __eq__ and other __[comparison]__ methods take any object, as they're the implementation of `img == a`, and `img == True` or `img == 1`. They're valid comparisons even if the other object isn't an image. False, but valid.